Source/ACE/Network/GameEvent/Events/GameEventStartBarber.cs

```diff
@@ -0,0 +1,51 @@
+﻿using ACE.DatLoader.FileTypes;
+using ACE.DatLoader.Entity;
+using ACE.DatLoader;
+using System;
+using ACE.Entity.Enum.Properties;
+using ACE.Entity;
+
+namespace ACE.Network.GameEvent.Events
+{
+    public class GameEventStartBarber : GameEventMessage
+    {
+        public GameEventStartBarber(Session session)
+            : base(GameEventType.StartBarber, GameMessageGroup.Group09, session)
+        {
+            // These are the motion tables for Empyrean float and not-float (one for each gender). They are hard-coded into the client.
+            const uint EmpyreanMaleMotionDID = 0x0900020Eu;
+            const uint EmpyreanFemaleMotionDID = 0x0900020Du;
+
+            // We will use this function to get the current player's appearance values.
+            AceCharacter characterClone = (AceCharacter)Session.Player.GetAceObject();
```

Comment:
This accesses the player.  If the player is modified while its being cloned it may crash the server.  Put this inside an ActionChain to mitigate that issue.